content/docs/user-guide/components/reference/terrain/world-renderer.md

```diff
@@ -11,6 +11,20 @@ The **Terrain World Renderer** component renders the terrain within the bounds s
 You must add this component to the *Level* entity, the parent of all entities in an **Open 3D Engine (O3DE)** level.
 {{< /important >}}
 
+## Usage 
+The terrain renderer component allows terrain to be visualized in the world and controls various global properties that apply to terrain everywhere. 
+
+#### Mesh Configuration
+These properties control the mesh density and blending of terrain LODs (level of detail) and have a very direct impact on vertex performance. The density of the first terrain
+LOD is set by the query resolution in the [Terrain World](/docs/user-guide/components/reference/terrain/world) component, and **First LOD distance** controls how far to render at this
+density. Each subsequent LOD will render for twice the distance at half the resolution all the way until **Mesh render distance** is met. This means that **First LOD distance** actually
+has a much greater impact on total triangle count than the total rendering distance.
+#### Detail Configuration
+These settings impact how far detail materials render, how they fade out, and how they blend with each other. Most of these settings have little impact on performance, however a larger
+render distance here will require more surface data points to be queried as you move around the world and also take more memory.
```

Comment:
Doesn't querying more surface data points directly impact the overall performance?

Reply:
Which is why I said 

> however a larger render distance here will require more surface data points to be queried as you move around the world and also take more memory.

So the performance is impacted _a little_ as you move the camera, but it's pretty negligible when not moving, and even overall it's not a huge cost to query 1000 surface points every few frames. Doesn't really compare to vertex cost of dense meshes.

Reply:
At a minimum, I'd suggest rephrasing this as "Most of these settings have little impact on per-frame rendering performance", since there is a noticeable performance impact when moving the camera right now.